subsys/dfu/boot/mcuboot.c

```diff
@@ -77,7 +77,9 @@ struct mcuboot_v1_raw_header {
 #define FLASH_AREA_IMAGE_SECONDARY FLASH_AREA_ID(image_1_nonsecure)
 #else
 #define FLASH_AREA_IMAGE_PRIMARY FLASH_AREA_ID(image_0)
+#if FLASH_AREA_LABEL_EXISTS(image_1)
```

Comment:
This probably causes problem with `smp_svr` overwriting itself. The compiler does not go inside the `#if`, probably due to the PM taking over the role of flash manager.

Reply:
I will look into it

Reply:
https://github.com/ioannisg/sdk-nrf/pull/2

Can you try this?

Reply:
@de-nordic can you confirm?